app/World.php

```diff
@@ -168,6 +180,51 @@ public function __construct(int $id = 0, array $config = [])
                 $this->config['item.count.TwentyRupees2'] = 4;
         }
 
+        switch ($this->config('item.functionality')) {
+            case 'superexpert':
+                $this->config['rom.CapeMagicUsage.Normal'] = 0x01;
+                $this->config['rom.CapeMagicUsage.Half'] = 0x02;
+                $this->config['rom.CapeMagicUsage.Quarter'] = 0x02;
+                $this->config['rom.CaneOfByrnaInvulnerability'] = false;
+                $this->config['rom.PowderedSpriteFairyPrize'] = 0x79; // bees
+                $this->config['rom.BottleFill.Health'] = 0x00; // nothing
+                $this->config['rom.BottleFill.Magic'] = 0x00; // nothing
+                $this->config['rom.CatchableFairies'] = false;
+                $this->config['rom.CatchableBees'] = true;
+                $this->config['rom.StunItems'] = 0x00;
+                $this->config['rom.SilversOnlyAtGanon'] = true;
+                $this->config['rom.NoFarieDrops'] = true;
+                break;
+            case 'expert':
+                $this->config['rom.CapeMagicUsage.Normal'] = 0x02;
+                $this->config['rom.CapeMagicUsage.Half'] = 0x04;
+                $this->config['rom.CapeMagicUsage.Quarter'] = 0x08;
+                $this->config['rom.CaneOfByrnaInvulnerability'] = false;
+                $this->config['rom.PowderedSpriteFairyPrize'] = 0xD8; // 1 heart
+                $this->config['rom.BottleFill.Health'] = 0x20; // 7 hearts, 1/2 magic refills
```

Comment:
I think the comments for hard/expert potions are swapped.

Reply:
They are!  Good catch